pkg/integration/README.md

```diff
@@ -43,32 +42,21 @@ The run step has four arguments passed in:
 
 Try to do as much setup work as possible in your setup step. For example, if all you're testing is that the user is able to resolve merge conflicts, create the merge conflicts in the setup step. On the other hand, if you're testing to see that lazygit can warn the user about merge conflicts after an attempted merge, it's fine to wait until the run step to actually create the conflicts. If the run step is focused on the thing you're trying to test, the test will run faster and its intent will be clearer.
 
-#### Assert after input
```

Comment:
This is no longer really needed, given that most of our methods now run their own assertions